pom.xml

```diff
@@ -207,13 +201,13 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.13.1</version>
+            <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>net.logstash.logback</groupId>
             <artifactId>logstash-logback-encoder</artifactId>
-            <version>5.2</version>
+            <version>8.0</version>
```

Comment:
I see that you updated major version in `logstash-logback-encoder` and it is used in the main program. Is there any breaking changes that effects the logs structure or anything else?